packages/ui/src/Customization/CustomizationsExtensionListItem.tsx

```diff
@@ -59,23 +59,23 @@ export class CustomizationsExtensionListItem extends React.Component<
     );
   }
 
-  public handleDelete = () => {
+  public handleDelete() {
```

Comment:
`this` here will not be what you'd expect to be since this method hasn't bind bound to the class instance in the constructor.